flyway-core/src/test/resources/migration/dbsupport/postgresplus/sql/procedure/~V2__Invalid.sql

```diff
@@ -0,0 +1,21 @@
+--
```

Comment:
As I mentioned in my initial comments, I just could not figure out why the Oracle tests included this "V2_Invalid.sql" test - it's essentially designed to fail, which fails the whole procedure test. I decided, against convention, to keep the file in the test scripts, rather than just deleting it - I was hoping someone looking at this PR could tell me one way or another why I might need it...